messaging/src/androidTest/java/io/lantern/messaging/metadata/MetadataTest.kt

```diff
@@ -43,6 +43,31 @@ class MetadataTest : BaseTest() {
         assertNotNull(md.thumbnail)
         assertTrue(md.thumbnail!!.size <= file.length())
         assertEquals("image/webp", md.thumbnailMimeType)
+        // test additionalMetadata - should exist
+        val additionalMetadata = md.additionalMetadata
+        assertNotNull(additionalMetadata)
+        // test rotation - should exist and be 0
+        val rotation = additionalMetadata["rotation"]
+        assertNotNull(rotation)
+        assertEquals("0", rotation)
+    }
+
+    @Test
+    fun testUpsideDownVideo() {
```

Comment:
You could reduce duplication between these methods by making a helper method that does all the testing and just takes an expected value for rotation.